apps/web/src/ApplicationReadyGuard.tsx

```diff
@@ -34,7 +34,7 @@ export function ApplicationReadyGuard({ children }: PropsWithChildren<{}>) {
   function isOnboardingComplete() {
     if (IS_EE_AUTH_ENABLED) {
       // TODO: replace with actual check property (e.g. isOnboardingCompleted)
-      return currentOrganization?.productUseCases !== undefined;
+      return currentOrganization?.productUseCases !== undefined || currentOrganization?.language !== undefined;
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** ‏ I'm thinking this is where we might want to consider getting this data from all of our users. If we change the logic to only check if the languages data is present, we can show the form to every user logging in. Question is if we want to. @SokratisVidros wdyt?

Reply:
Correct, if we remove the language check, the form would re-appear BUT it will reappear for the first user that gets logged in, which might not be necessary an org owner and the user might just click anything to make the from disappear (speaking from personal experience). I assume the owner usually has better incentive to fill it correctly upon org creation.

Reply:
Same applies when a user invited, or auto joins an organization. For now, org level should be enough.

Reply:
if this stays on the org level then my question doesn't make sense. We can agree to keep it this way and move on